exercises/space-age/canonical-data.json

```
@@ -80,6 +80,16 @@
         "seconds": 1821023456
       },
       "expected": 0.35
+    },
+    {
+      "uuid": "57b96e2a-1178-40b7-b34d-f3c9c34e4bf4",
+      "description": "age on Sun",
```

If I'm not mistaken, "sun" being a common noun needs a definitive article when we're talking about our sun in our solar system.
```suggestion
      "description": "age on the Sun",
```